### PR TITLE
*: use coreos/go-systemd/activation for socket activation

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
fixed godep

File: start.go [hunk 1]
```diff
@@ -124,12 +119,9 @@ func startContainer(context *cli.Context, spec *specs.LinuxSpec, rspec *specs.Li
 	process := newProcess(spec.Process)
 	// Support on-demand socket activation by passing file descriptors into the container init process.
 	if os.Getenv("LISTEN_FDS") != "" {
```
Comment (crosbymichael) [inline start.go]: I think you can do everything here and not have the same call above and below

Comment (runcom) [inline start.go]: That was my thought as well but didn't want to break what already done

Comment (crosbymichael) [inline start.go]: might as well fix it now

**Comment by runcom** [comment]
@crosbymichael fixed

**Comment by crosbymichael** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**